Aptoide/aptoide-client-v8
# [feature/MOB-223] Search UX Ads

**What does this PR do?**

   This PR attempts to improve the user experience of ads in search according to what is specified in the ticket. Some things include putting the banner in the bottom, removing legacy ads, downsizing the viewholder etc. See the ticket for full specification.

**Database changed?**

   No

**How should this be manually tested?**

  Check if it follows everything that was specified in the ticket, including the new (reduced) card design, the banner ad and the native mopub ad and check if the display correctly. Also check the that mopub keys are implemented as specified.

**What are the relevant tickets?**

  [MOB-223](https://aptoide.atlassian.net/browse/MOB-223)

**Code Review Checklist**
